libethcore/Transaction.h

```diff
@@ -14,19 +14,16 @@
 	You should have received a copy of the GNU General Public License
 	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
 */
-/** @file TransactionBase.h
- * @author Gav Wood <i@gavwood.com>
- * @date 2014
- */
 
 #pragma once
 
+#include <boost/optional.hpp>
```

Comment:
`#include`s should be in increasing order of generality, so this goes after our own headers

Reply:
I think we have it the other way around in most files. Personally, I don't care. Why is this better?

Reply:
Link from coding standards: http://stackoverflow.com/questions/614302/c-header-order/614333#614333

It prevents the headers from not including all necessary headers but still working because of weird inter-header dependency